examples/nlp/README.md

```diff
@@ -0,0 +1,3 @@
+# NLP Inference Examples
```

Comment:
This can be written as Natural Language Processing (NLP) Inference Examples

Reply:
done